app/views/trainee/user_courses/_user_course.html.erb

```diff
@@ -0,0 +1,11 @@
+<li class="<%= status_user_course user_course %>">
+  <%= link_to user_course.course.name, trainee_user_course_path(user_course.id) %>
```

Comment:
`user_course.course.name` tìm hiểu về degelate in rails hì

Reply:
Hết mấy cái filed của course rồi đó chị. Còn mỗi cái này
<%= user_course.course.subjects.size %>